core/MySensorsCore.cpp

```diff
@@ -355,9 +355,20 @@ bool present(const uint8_t childSensorId, const uint8_t sensorType, const char *
              const bool ack)
 {
 	return _sendRoute(build(_msgTmp, GATEWAY_ADDRESS, childSensorId, C_PRESENTATION, sensorType,
-	                        ack).set(childSensorId==NODE_SENSOR_ID?MYSENSORS_LIBRARY_VERSION:description));
+	                        ack).set(childSensorId == NODE_SENSOR_ID ? MYSENSORS_LIBRARY_VERSION : description));
 }
 
+#if !defined(__linux__)
```

Comment:
Is there no clean way to avoid the code duplication? I worry that a change in one of these nearly identical functions might be done without making the corresponding change to the other function.

Reply:
The magic happens in MyMessage::set() - I do not see how this can be improved.

Reply:
ok